client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -82,6 +83,16 @@
      * @param checkpoint The checkpoint to restore to.
      */
     void resetReadersToCheckpoint(Checkpoint checkpoint);
+
+    /**
+     * Restore the reader group to the provided {@link StreamCut StreamCut(s)}. All readers in the group will encounter
```

Comment:
This should require (and document) that a full set of streamCuts are needed based on the readerGroup's current configuration. IE: You can't simply leave one out to not consume from that stream. Nor can you add one to add a new stream.

Reply:
done.